### ISSUE TITLE
Mac OS: Limited access to Nextcloud client via Voiceover

### ISSUE THREAD

**Issue by caturria** [nextcloud/desktop#1730]
### Expected behaviour
All options within the Nextcloud client interface should be accessible to a blind user via Voiceover and the keyboard.

### Actual behaviour
Voiceover can access a very limited selection of the options, gets stuck inside the file list and cannot be used to make configuration changes to the client.

### Steps to reproduce
1. Install Nextcloud client on any Mac, connect to a server and sync some files.
2. Turn on Voiceover by pressing Command+F5.
3. Once you hear speech, press Control+Option+M twice quickly to bring up Apple's menu extras list.
4. Press Control+Option+Right until you hear Nextcloud.
5. Press Control+Option+Space. Observe that this should bring up the Nextcloud menu, but it does not. To work around it, perform a right mouse click. If it doesn't work, you will have to go to Applications > Utilities > Voiceover Utility and configure the mouse cursor to follow the VoiceOver cursor before repeating steps 1 - 5 again.
6. Press Control+Option+Down until you hear Voiceover say "Settings..." Press Control+Option+Space to open the settings window.
7. Use Control+Option+Left and Control+Option+Right to scroll through the options.
Observe that the available options are:
<ul>
<li>Close button</li>
<li>Minimize button</li>
<li>Zoom button</li>
<li>Nextcloud</li>
<li>User@domain.tld</li>
<li>Activity, check box</li>
<li>A button with no label to tell a screen reader what its function is</li>
<li>General, checkbox</li>
<li>Network, checkbox</li>
<li>The encryption details</li>
<li>Connected to https://domain.tld as Username</li>
<li>An 'Account' button which does not reveal anything to Voiceover when interacted with using Control+Option+Space</li>
<li>75GB in use</li>
<li>Next is the first file name and its size, but Voiceover gets stuck here. Continuing to press Control+Option+Right will cause a broken record effect wherein the name of the first file and its size will be announced in a loop.</li>
</ul>
Note: checking and unchecking the 'Activity', 'General' or 'Network' checkboxes does not reveal any additional options.
For this reason I do not know all of the available features of this application, but I do know that a Voiceover user such as myself cannot access the interface to exclude certain files and folders from syncing, among other things.

### Client configuration
Client version:
2.6.2

Operating system:
Mac OS Catalina
OS language:
English
Qt version used by client package (Linux only, see also Settings dialog):

Client package (From Nextcloud or distro) (Linux only):

Installation path of client:
/Applications

### Server configuration

Nextcloud version:
17.0.2
Storage backend (external storage):
Local storage
### Logs

Please use Gist (https://gist.github.com/) or a similar code paster for longer
logs.

1. Client logfile: Output of `nextcloud --logwindow` or `nextcloud --logfile log.txt`
(On Windows using `cmd.exe`, you might need to first `cd` into the Nextcloud directory)
(See also https://docs.nextcloud.com/desktop/2.3/troubleshooting.html#log-files)
N/A
2. Web server error log:
N/A
3. Server logfile: nextcloud log (data/nextcloud.log):
N/A

**Comment by Rello** [comment]
The client has improved a lot since the ticket was opened.
If it is still relevant, please create a fresh ticket.
Thank you for your support
